app/scripts/components/analysis/define/constants.ts

```diff
@@ -0,0 +1,22 @@
+import { Feature, Polygon } from 'geojson';
+
+export type RegionPreset = 'world';
+
+export const FeatureByRegionPreset: Record<RegionPreset, Feature<Polygon>> = {
+  world: {
+    type: 'Feature',
+    properties: {},
+    geometry: {
+      coordinates: [
+        [
+          [-180, -89],
+          [180, -89],
+          [180, 89],
+          [-180, 89],
+          [-180, -89]
+        ]
+      ],
+      type: 'Polygon'
+    }
+  }
```

Comment:
To work with the AOI editor, the feature needs an `id`, otherwise if you try to edit it, the app will crash